lumibot/backtesting/polygon_backtesting.py

```diff
@@ -223,10 +221,7 @@ def get_historical_prices_between_dates(
         start_date=None,
         end_date=None,
     ):
-        pandas_data_update = self.update_pandas_data(asset, quote, 1, timestep)
-        if pandas_data_update is not None:
-            # Add the keys to the self.pandas_data dictionary
-            self.pandas_data.update(pandas_data_update)
+        self.update_pandas_data(asset, quote, 1, timestep)
```

Comment:
why is update_data_store=False here? shouldnt we also update the data store here if needed?

Reply:
I don't know why only pandas_data is updated in three places and _data_store is updated in one.